extensions/amp-story/1.0/amp-story.js

```diff
@@ -402,6 +402,7 @@ export class AmpStory extends AMP.BaseElement {
     }
     if (
       isExperimentOn(this.win, 'story-disable-animations-first-page') ||
+      !this.storeService_.get(StateProperty.CAN_ANIMATE_FIRST_PAGE) ||
```

Comment:
The store service should always have the correct and up to date value, here this state would say `true` even when the experiment on the line above is enabled

Reply:
Are you saying this should be something like
```js
if (isExperimentOn(this.win, 'story-disable-animations-first-page') {
  this.storeService.set(StateProperty.CAN_ANIMATE_FIRST_PAGE, false)
}
```
then always check the store value?

This means we are ok with the experiment val overriding the `embedMode` default?

Reply:
No we don't need to change the UX, but the value from the store service should always be the one source of truth.
We should remove all other `isExperimentOn()` and `prefersReducedMotion` checks and only rely on the store service, which is why it should always be up to date with the right value.

Reply:
Changing this as well.